tests/views_test.py

```diff
@@ -209,6 +209,7 @@ def test_region_with_all_optional_parameters(self) -> None:
         assert region.lod.min_fade_extent == 0
         assert region.lod.max_fade_extent == 512
         assert region
+        assert region.lod.__bool__() is True
```

Comment:
I think `assert region`  does the same as 
```suggestion
        assert bool(region)
```